server/src/db/Seeder.js

```diff
@@ -5,11 +5,17 @@ import { User } from '../models/index.js'
 class Seeder {
   static async seed() {
 
+    console.log("seeding users")
     await User.query().insert({
       email: 'test@test.com',
       cryptedPassword: 'hithere',
       username: 'test'
     })
+    await User.query().insert({
+      email: 'test2@test.com',
+      cryptedPassword: 'hithere',
+      username: 'test2'
+    })
```

Comment:
i would stick with the pattern you have for other seeders: use a UserSeeder

Reply:
Resolved this in the branch we've been working on today that's based off of this branch so I'm going to leave it for now to reduce conflicts.